### ISSUE TITLE
Investigate and fix tabIndex adding cause and behaviour

### ISSUE THREAD

**Issue by nhsbsa-mamcp** [nhsbsa-data-analytics/care-home-prescribing-2020-2023#66]
Tab indexes are being added by something not in our code (I think), causing accessibility fail.
